packages/ee/shared/src/lib/custom-domains/index.ts

```diff
@@ -18,6 +18,13 @@ export type CustomDomain = Static<typeof CustomDomain>;
 
 export const AddDomainRequest = Type.Object({
     domain: Type.String(),
+    ssl: Type.Object({
```

Comment:
Let's remove this as we will provide static one in the backend same as cloudflare user interface